src/styles/components/_about-us-section.scss

```diff
@@ -20,11 +20,16 @@
     display: flex;
     justify-content: center;
 
-    &--quote {
+    &--quote,
+    &--goal {
```

Comment:
No need to duplicate this, how about creating a single modifier which the name fits the action? (The action is to hide in small screen, so think of a name for it instead of having two modifiers doing the same thing. **Modifier** is to modify => modify things differently.)